src/components/AppHeader.vue

```diff
@@ -1,63 +1,79 @@
 <template>
-	<!-- TODO: Update to ANTD menu and header -->
-	<div class="header" id="header">
-		<a-row class="veritcal_center">
-			<a-col :span="20">
-				<a-row class="veritcal_center">
-					<a-col :span="4">
-						<h1>SigmaDao</h1>
-					</a-col>
-					<a-col :span="8">
-						<div class="menu_id">
-							<label for="dao_id" style="padding-right: 10px">DAO ID</label>
-							<input id="dao_id" v-model="id" type="number" />
-						</div>
-					</a-col>
-					<a-col>
-						<div>
-							<router-link :to="{ path: '/vote' }">
+	<div class="components-page-header-demo-responsive">
+		<a-page-header>
+			<template #title>
+				<img src="../assets/logo.png" class="logo" />
+			</template>
+			<template #extra>
+				<WalletConnect />
+			</template>
+			<div class="content">
+				<div class="main">
```

Comment:
why do we have 3 nested divs (components-page-header-demo-responsive, content, main) + <a-page-header>?
Can we reduce it to 2?

So, basically we have 4 elements without anything in the middle, slike:
```
<div>  // components-page-header-demo-responsive
  <div>  // a-page-header
    <div>  // content
      <div> //main

         real content

      <div>
    <div>
  <div>
<div>
```